<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bug Localization</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="analyse">
          <property name="text">
           <string>analyse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>add testcase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhHiddenText</set>
          </property>
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="run">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextEditWithLineNum" name="qtCodeArea">
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QTextEditWithLineNum" name="qtElementArea">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="qtMatrixArea">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>25</number>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="separator"/>
    <addaction name="actionopen_code"/>
   </widget>
   <widget class="QMenu" name="menumethod">
    <property name="title">
     <string>method</string>
    </property>
    <addaction name="actiondefault"/>
    <addaction name="actiondstar"/>
    <addaction name="actionochiai"/>
    <addaction name="actionbarinel"/>
    <addaction name="actionTarantula"/>
   </widget>
   <addaction name="File"/>
   <addaction name="menumethod"/>
  </widget>
  <action name="actionopen">
   <property name="text">
    <string>open...</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save...</string>
   </property>
  </action>
  <action name="actionopen_code">
   <property name="text">
    <string>code  open...</string>
   </property>
  </action>
  <action name="actiondefault">
   <property name="text">
    <string>default</string>
   </property>
  </action>
  <action name="actiondstar">
   <property name="text">
    <string>dstar</string>
   </property>
  </action>
  <action name="actionochiai">
   <property name="text">
    <string>ochiai</string>
   </property>
  </action>
  <action name="actionbarinel">
   <property name="text">
    <string>barinel</string>
   </property>
  </action>
  <action name="actionTarantula">
   <property name="text">
    <string>Tarantula</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTextEditWithLineNum</class>
   <extends>QTextEdit</extends>
   <header location="global">qtexteditwithlinenum.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="source/source.qrc"/>
 </resources>
 <connections/>
</ui>
